main.go

```diff
@@ -1,2 +1,49 @@
 package main
 
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+func main() {
+
+	secret, err := os.ReadFile("botID.txt")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	sess, err := discordgo.New(string(secret))
```

Comment:
We will want to interact with secrets in a better fashion later down the road but for now this should be fine. 👍